### PR TITLE
Build: Less child-process spawning, migrate away from execa, pure `child_process` use instead.

### PR DESCRIPTION
**Pull request by ndelangen**
- start removing `execa` and using `child_process` directly
- do not spawn verdaccio in a sub-process, but using existing code
- do not spawn `wait-on` in a sub-process, but use node-api
- use `lerna` defined in `./scripts` rather then in `./code`
- use locally installed version of lerna instead of getting it from `npx`
- use locally installed version of chromatic instead of getting it from `npx`
- make chromatic exit early after upload, saving CircleCI computation time
- decrease verdaccio log noise

### CODE REVIEW

**Comment by tmeasday** [comment]
Have you tried running the chromatic process with `--exit-once-uploaded` locally and seeing what happens? It seems suspicious that it still takes the same 3.5mins that it took without the flag. Doesn't seem like an execa problem (it would just timeout wouldn't it?).

**Comment by ndelangen** [comment]
@tmeasday The CLI output does suggest that the flag had an effect.

I assumed it was something weird execa related, we've seen issues with execa before.
So I tried to eliminate that possibility.

**Comment by ndelangen** [comment]
The CI still doesn't exit any earlier after migrating away from execa

**Comment by ndelangen** [comment]
We've run into issues where tasks processes didn't exit correctly.
I still assume it's due to complexity and `execa`.

Turns out the long halting of the chromatic task is something recently introduced in the chomatic-cli, but my investigation into it yielded this PR.

File: scripts/tasks/chromatic.ts [hunk 1]
```diff
@@ -8,19 +8,17 @@ export const chromatic: Task = {
...
-      { dryRun, debug }
-    );
+    const command = `yarn chromatic`;
+    const flags = [
+      `--storybook-build-dir=${builtSandboxDir}`,
+      `--exit-once-uploaded`,
+      `--projectToken=${token}`,
+    ];
+
+    await exec(`${command} ${flags.join(' ')} `, { cwd: scriptsDir }, { dryRun, debug });
```
Comment (tmeasday) [inline scripts/tasks/chromatic.ts:22]: We could actually use the chromatic node API for this now 🤷

**Review by tmeasday — APPROVED** [review]
LGTM, seems like a step forward. Maybe the title of this PR is a little out of date?

File: scripts/tasks/chromatic.ts (hunk 1 above)
Comment (ndelangen) [reply scripts/tasks/chromatic.ts:22]: Yes, if it's stable, we should, I think.

**Pull request closed without merging**